ci/pipeline.yml

```diff
@@ -59,6 +60,61 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ((aws-rds-secret-access-key))
       GATEWAY_HOST: http://prometheus-production.service.cf.internal
 
+- name: aws-iam-check-keys
+  serial_groups: [production]
+  plan:
+  - in_parallel:
+    - get: iam-keys-check-timer
+      trigger: true
+    - get: prometheus-config
+      passed: [set-self]
+    - get: aws-admin
+      trigger: true
+    - get: terraform-yaml-production
+      trigger: true
+    - get: terraform-prod-com-yml
+      resource: terraform-prod-com-yml
+      trigger: true
+    - get: terraform-prod-gov-yml
+      resource: terraform-prod-gov-yml
+      trigger: true
+    - get: other-iam-users-yml
+      resource: other-iam-users-yml
+      trigger: true
+  - task: aws-iam-check-keys
+    file: prometheus-config/ci/aws-iam-check-keys.yml
+    tags: [iaas]
+    params:
+      AWS_DEFAULT_REGION: ((aws-region))
+      IAM_KEYS_DB: ((aws-iam-keys-db))
+      IAM_KEYS_USER: ((aws-iam-keys-user))
+      IAM_KEYS_PASSWORD: ((aws-iam-keys-password))
+      IAM_KEYS_HOST: ((aws-iam-keys-host))
+      IAM_KEYS_PORT: ((aws-iam-keys-port))
+      IAM_CREATE_TABLES: ((aws-iam-create-tables-bool))
+      GATEWAY_HOST: prometheus-staging.service.cf.internal
+      PREFIX_DELIMITER: ((aws-iam-prefix-delimiter))
+      WARN_DAYS: ((aws-iam-warn-days))
+      VIOLATION_DAYS: ((aws-iam-violation-days))
+  on_success:
+    put: slack
+    params:
+      text: |
+        :white_check_mark: Successfully ran IAM Check Keys in production
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+      channel: ((slack-channel-success))
+      username: ((slack-username))
+      icon_url: ((slack-icon-url))
+  on_failure:
+    put: slack
+    params:
+      text: |
+        :x: FAILED to deploy IAM Check Keys on production
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+      channel: ((slack-channel-success))
```

Comment:
is `slack-channel-success` correct here?

Reply:
That's what Van asked me to use, so yes.